AUTHORS

```diff
@@ -73,5 +74,3 @@ and moral support from (alphabetically by first name or nickname):
  - vtriolet
  - Walter Lapchynski
  - Waneon Kim
-
- > This list was updated to revision 6e8d820737dea0f3e08f12b10768facef19be684 on May 28th 2022.
```

Comment:
I would prefer doing this in a separate merge, when it's actually a complete update (e.g. removes BlueSystems mention, adds all authors since that commit, updates the stamp).